infra/cifuzz/fuzz_target.py

```diff
@@ -109,7 +112,8 @@ def fuzz(self):
         'bash', '-c'
     ]
     run_fuzzer_command = 'run_fuzzer {fuzz_target} {options}'.format(
-        fuzz_target=self.target_name, options=LIBFUZZER_OPTIONS)
+        fuzz_target=self.target_name,
+        options=LIBFUZZER_OPTIONS + ' -max_total_time=' + str(self.duration))
```

Comment:
nit: unrelated to this CL, but rather than passing command line flags as a string, which has issues with modifications and quoting, we should pass a list instead.